api/response/schema_query.py

```diff
@@ -154,7 +187,8 @@ def search_response_by_room(
 
     @Auth.user_roles('Admin')
     def resolve_all_room_responses(
-            self, info, lower_limit=None, upper_limit=None, room=None):
+            self, info, page=None, per_page=None, lower_limit=None,
+            upper_limit=None, room=None, **kwargs):
         responses = Query.get_all_reponses(self, info)
```

Comment:
This variable `responses` on line 192 is not being used. You can remove it.

Reply:
Thanks @kampkelly, I've updated this.

Reply:
Okay, cool.